### ISSUE TITLE
About DTU dataset

### ISSUE THREAD

**Issue by 05063112lcs** [TQTQliu/ET-MVSNet#19]
Hello, I have some questions when I reproduce your work, and I hope you can help answer them.
1. After completing the training of the DTU dataset, I tested it in the test set, and the acc, comp, and all indicators were the same as in the article, but there was noise in the obtained point cloud data, so I referred to your reply to others and changed the relevant part of the test_dypcd.py file to photometric_confidence = confidence_list[0]*confidence_list[1]* confidence_list[2]*confidence_list[3], but after evaluation, it was found that although the noise of the point cloud data is much less, the relevant data indicators are much worse than the original text, can you help with this?
2. For the data in your paper, whether the point cloud data with noise removal is used for evaluation？
![image](https://github.com/TQTQliu/ET-MVSNet/assets/149451618/96aa8f4b-36a1-4ad0-95db-156357a959ac)
![image](https://github.com/TQTQliu/ET-MVSNet/assets/149451618/b9f00573-501a-4f35-ac2e-175441fbabc7)

**Comment by TQTQliu** [comment]
1. When changed to `photometric_confidence = confidence_list[0]confidence_list[1] confidence_list[2]*confidence_list[3]`, you should change the confidence threshold as well.
2. No noise removal was used for point cloud data.

**Comment by 05063112lcs** [comment]
There are still some questions :1. The evaluation indicators are consistent with those described in your paper, can it mean that I have successfully reproduced, despite the noise in my point cloud data? 2. As for the threshold you just mentioned, I also saw it in your reply to others, but I still don't know which parameter should be modified, please kindly give me some guidance, thank you!



---Original---
From: "Tianqi ***@***.***&gt;
Date: Mon, Jul 1, 2024 19:13 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About DTU dataset (Issue #19)




  
When changed to photometric_confidence = confidence_list[0]confidence_list[1] confidence_list[2]*confidence_list[3], you should change the confidence threshold as well.
 
No noise removal was used for point cloud data.

**Comment by TQTQliu** [comment]
1.yes
2. modify the `--conf` in the [script](https://github.com/TQTQliu/ET-MVSNet/blob/e29d0b6c1d7772a86b730f4b8b106df3846a7fb2/scripts/test_dtu.sh#L19). When changed to photometric_confidence = confidence_list[0]confidence_list[1] confidence_list[2]*confidence_list[3], the confidence threshold `--conf` should be smaller.

**Comment by 05063112lcs** [comment]
I see a test dtu pbd.sh file in the code you distribute. This script file is not used in the entire reproduction process. Instead, I use the test dtu.sh file. I was testing on TNT data set today. Is the testing process of tnt adv very slow? My testing process has been almost 10h and has not been completed yet. On the contrary, tnt inter has been finished.



---Original---
From: "Tianqi ***@***.***&gt;
Date: Mon, Jul 1, 2024 19:31 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] About DTU dataset (Issue #19)




 
1.yes
 2. modify the --conf in the script. When changed to photometric_confidence = confidence_list[0]confidence_list[1] confidence_list[2]*confidence_list[3], the confidence threshold --conf should be smaller.

**Comment by TQTQliu** [comment]
Yes, it takes longer, but you can see some intermediate results.

**Comment by 05063112lcs** [comment]
Hello, I modified the conf indicator according to what you said, and the final acc, comp, and all indicators are very close to your paper, but the qualitative analysis results are still very different from those in your paper, no matter how small the settings are, there is still a big difference between them and your paper, I haven't changed your code, I don't know what the reason is, can you help solve it, the GPU I use is A5000, or the evaluation of the DTU dataset needs to run test
-pcd_dtu.sh instead of test_dtu.sh when I set the conf indicator to 0.1, the effect is as follows.
![image](https://github.com/TQTQliu/ET-MVSNet/assets/149451618/173aa43d-c41e-41f2-ab78-5809c1e06e74)
![image](https://github.com/TQTQliu/ET-MVSNet/assets/149451618/3285bdd8-2913-431f-941e-923e0f3aab02)

**Comment by wsh1057** [comment]
你好，请问0.2924综合指标你是怎么复线的 为什么我按照作者readme的步骤进行复现得到的综合指标是0.3424，我不知道哪里出了问题
Hello, could you please explain how you reproduced the 0.2924 composite metric? I followed the steps in the author's README but obtained a composite metric of 0.3424 instead. I'm not sure where I went wrong.
